enforcer-rules/README.md

```diff
@@ -0,0 +1,89 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule verifies that the transitive dependency tree of `pom.xml` does not have
+any [linkage error](../library-best-practices/glossary.md#types-of-conflicts-and-compatibility).
```

Comment:
errors (plural)

Reply:
"Any" is followed by a singular noun.

Reply:
Yes, but that singular noun refers to a specific member of a group rather than the group a whole. 

Thus

 "If you have any trouble, please let me know,"

but 

 "If you have any problems, please let me know,"